pr2eus/robot-interface.l

```diff
@@ -401,7 +401,15 @@
    av)
   (:angle-vector-sequence
    (avs &optional (tms (list 3000)) (ctype controller-type) (start-time 0.1) &key (scale 1) (min-time 0.0) (end-coords-interpolation nil))
-   "Send joind angle to robot, this method retuns immediately, so use :wait-interpolation to block until the motion stops."
+   "Send joind angle to robot, this method retuns immediately, so use :wait-interpolation to block until the motion stops.
+- asv : list of joint angle vector [rad]
```

Comment:
I see, so have to change https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/035f676471e9ff0748f2a0199dbe930685b1c04e/pr2eus/robot-interface.l#L350 too